wodles/aws/aws_s3.py

```diff
@@ -3044,6 +3058,23 @@ def arg_valid_regions(arg_string):
     return final_regions
 
 
+def arg_valid_iam_role_duration(duration: int or None, iam_role_arn: str or None):
```

Comment:
```suggestion
def arg_valid_iam_role_duration(arg_string):
```

Argument validation functions will receive only one parameter when called